test/onboarding.js

```diff
@@ -1,58 +1,195 @@
 import test from 'ava';
 import { useSpectron, focusMain, focusChild } from './helpers/spectron/index';
-import { selectSource, clickSourceProperties } from './helpers/spectron/sources';
+import {
+  selectSource,
+  clickSourceProperties
+} from './helpers/spectron/sources';
+import {
+  openFiltersWindow,
+  closeFilterProperties
+} from './helpers/spectron/filters';
+import { getFormInput, setFormDropdown } from './helpers/spectron/forms';
+import {
+  getInputValue,
+  getInputValueId,
+  getInputCheckboxValue
+} from './helpers/spectron/advancedAudioSettings';
 
 useSpectron({ skipOnboarding: false });
 
 test('Adding some starter widgets', async t => {
   const app = t.context.app;
   await focusMain(t);
-
   const widgetToken = 'SomeWidgetToken';
   const platform = {
     type: 'twitch',
     username: 'exampleuser',
     token: 'SomeToken',
     id: 'SomeId'
   };
-
   // Wait for the auth screen to appear
   await app.client.isExisting('button=Twitch');
-
   await app.webContents.send('testing-fakeAuth', {
     widgetToken,
     platform
   });
-
   // This will only show up if OBS is installed
   if (await t.context.app.client.isExisting('button=Start Fresh')) {
     await t.context.app.client.click('button=Start Fresh');
   }
-
   // Select and deselect some widgets
-
   await app.client.click('div=Event List');
   await app.client.click('button=Remove Widget');
-
   await app.client.click('div=Chatbox');
   await app.client.click('button=Add Widget');
-
   await app.client.click('div=Donation Goal');
   await app.client.click('button=Add Widget');
-
   await app.client.click('button=Add 4 Widgets');
   await app.client.click('a=Setup later');
-
   t.true(await app.client.isExisting('li=Alert Box'));
   t.false(await app.client.isExisting('li=Event List'));
   t.true(await app.client.isExisting('li=The Jar'));
   t.true(await app.client.isExisting('li=Chat Box'));
   t.false(await app.client.isExisting('li=Donation Ticker'));
   t.true(await app.client.isExisting('li=Donation Goal'));
-
   await selectSource(t, 'Alert Box');
   await clickSourceProperties(t);
   await focusChild(t);
-
   t.true(await app.client.isExisting('label=Widget Type'));
 });
+
+test('Obs-importer', async t => {
+  const path = require('path');
+  const sh = require('shelljs');
+
+  const app = t.context.app;
+  const zipExe = path.resolve( __dirname, '..', '..', 'node_modules', '7zip-bin-win', 'x64', '7za.exe');
```

Comment:
Now that we're compiling test files, we should stop using `__dirname`.  It will refer to the location of the compiled file, not the location of the original source file (this file).  It looks like the CWD of the tests should be the root of the repo, so lets just rely on that fact.  This can just be:
```
path.resolve('node_modules', '7zip-bin-win', 'x64', '7za.exe');
```

Reply:
Done.